main/lsp/LSPTypechecker.cc

```diff
@@ -175,16 +185,17 @@ TypecheckRun LSPTypechecker::runSlowPath(LSPFileUpdates updates, bool isCancelab
     }
     logger->debug("Taking slow path");
 
-    UnorderedSet<int> updatedFiles;
-    vector<ast::ParsedFile> indexedCopies;
     vector<core::FileRef> affectedFiles;
     auto finalGS = move(updates.updatedGS.value());
     // Replace error queue with one that is owned by this thread.
     finalGS->errorQueue = make_shared<core::ErrorQueue>(finalGS->errorQueue->logger, finalGS->errorQueue->tracer);
     finalGS->errorQueue->ignoreFlushes = true;
     // Note: Commits can only be canceled if this edit is cancelable, LSP is running across multiple threads, and the
     // cancelation feature is enabled.
-    const bool committed = finalGS->tryCommitEpoch(updates.versionEnd, isCancelable, [&]() -> void {
+    const bool committed = finalGS->tryCommitEpoch(updates.versionEnd, cancelable, [&]() -> void {
```

Comment:
Naming question:  has anything actually been committed at this point (if this is true), or is this really more like `shouldCommit`, and committing actually begins in the `if` statement on line 261, below?

Reply:
Oh, that's a good question. It's really more like `shouldCommit`. I can rename it later.